testing/tests/DevExpress.ui.widgets.dataGrid/exportController.tests.js

```diff
@@ -1288,7 +1288,7 @@ QUnit.module('ExportController', {
         this.clock.tick(10);
 
         // assert, act
-        assert.equal(dataProvider.getCellData(0, 2).value, 'Max: 93% \n Min: 1');
+        assert.equal(dataProvider.getCellData(0, 2).value, 'Max: 93% n Min: 1');
```

Comment:
Look like a typo, let's return ```\n``` or remove it:

```suggestion
        assert.equal(dataProvider.getCellData(0, 2).value, 'Max: 93% \n Min: 1');
```

or

```suggestion
        assert.equal(dataProvider.getCellData(0, 2).value, 'Max: 93%  Min: 1');
```